services/server/src/lib/assets.ts

```diff
@@ -3,6 +3,8 @@ import { Request } from "express";
 import { isArray } from "lodash";
 import { v4 as uuid } from "uuid";
 import multer from "multer";
+import bytes from "bytes";
+import { logger } from "@litespace/utils";
```

Comment:
We don't use this logger. It is deprecated. Just add console.error or throw and error